integration-tests/__tests__/__snapshots__/globals.test.js.snap

```diff
@@ -20,6 +20,79 @@ Ran all test suites.
 "
 `;
 
+exports[`cannot test with no implementation 1`] = `
+"FAIL __tests__/only-constructs.test.js
+  ● Test suite failed to run
+
+    Missing second argument. It must be a callback function.
+
+      431 |       }
+      432 |       if (fn === undefined) {
+    > 433 |         throw new Error(
```

Comment:
Ugh, any way we can disable code frame for this one, @SimenB?

Reply:
It points to the `it` in real code, not sure why it shows the `throw` here..

![image](https://user-images.githubusercontent.com/1404810/36295731-a1fd313a-12e9-11e8-88be-ab926687bf61.png)

![image](https://user-images.githubusercontent.com/1404810/36295732-aaf0c4dc-12e9-11e8-93ff-6a9daf0400b7.png)

![image](https://user-images.githubusercontent.com/1404810/36295739-b48e5860-12e9-11e8-9664-d7cc385045d2.png)

Reply:
the frame is probably unable to find `__tests__/only-constructs.test.js` for whatever reason - I'd debug that

Reply:
Ok, I’ll look in it. Thanks!

Reply:
Ok, I’ll look in it. Thanks!

Reply:
It was pulling in the wrong frame because In jest-message-util 

```const getTopFrame = (lines: string[]) => {
  for (const line of lines) {
    if (
      line.includes(PATH_NODE_MODULES) ||
      line.includes(PATH_EXPECT_BUILD)
    ) {
      continue;
    }
```

Wouldn't catch if the the top frame was from jest packages, so I added a condition:

```
const PATH_JEST_PACKAGES = `${path.sep}jest${path.sep}packages${path.sep}`;
```
that way it ignores the jest code as the topframe.